README.md

```diff
@@ -1,6 +1,7 @@
 [![Lighthouse PWA Test](https://github.com/ivelin/olympus-frontend/actions/workflows/lighthouse.yml/badge.svg)](https://github.com/ivelin/olympus-frontend/actions/workflows/lighthouse.yml)
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](code_of_conduct.md)
 [![OHM Discord](https://img.shields.io/badge/chat-on%20discord-7289DA.svg)](https://discord.gg/gGZUMVDuhQ)
+[![Coverage Badge](https://img.shields.io/endpoint?url=https://gist.githubusercontent.com/midnight-madman/4d79fbc5bcabae6ded617354b1896066/raw/olympus-frontend__coverage__heads_develop.json)]
```

Comment:
@midnight-madman this looks great.
Would you be able to show all 4 coverage metrics?
`branch coverage` is probably the most valuable one to keep an eye on.
Thank you!